ui/preferences/src/main/java/de/danoeh/antennapod/ui/preferences/screen/downloads/DataFolderAdapter.java

```diff
@@ -23,13 +24,13 @@ public class DataFolderAdapter extends RecyclerView.Adapter<DataFolderAdapter.Vi
     private final Consumer<String> selectionHandler;
     private final String currentPath;
     private final List<StoragePath> entries;
-    private final String freeSpaceString;
+    private final Resources resources;
 
     public DataFolderAdapter(Context context, @NonNull Consumer<String> selectionHandler) {
         this.entries = getStorageEntries(context);
         this.currentPath = getCurrentPath();
         this.selectionHandler = selectionHandler;
-        this.freeSpaceString = context.getString(R.string.choose_data_directory_available_space);
+        this.resources = context.getResources();
```

Comment:
Storing this in a variable was an (unnecessary) performance improvement. Now that we need to load the plural anyway, I don't think we need to store the resources object there. We can just directly call `context.getResources` below when binding the view.